.env

```diff
@@ -16,4 +16,7 @@ VITE_WEBSITE_EMAIL=PCASKIN@CPSkinHealth.com
 VITE_NEW_WEBSITE=https://new.pcaskinpro.com/
 
 #AuthAPI URL or login page URl
-VITE_AUTH_URL=https://be-colgate.onrender.com/api
\ No newline at end of file
+VITE_AUTH_URL=https://be-colgate.onrender.com/api
+
+VITE_PRODUCT_IMAGE_URL=https://www.pcaskinpro.com/cms/assets/846329ca-d3ee-4e69-8e8c-3d558404b6aa.jpg?key=pca-en-us-marketing-header-large
```

Comment:
Already we have defined VITE_SCROLL_PCA_CMS_URL use this instead of creating new one https://www.pcaskinpro.com/cms/assets